hawkbit-ddi-resource/src/main/java/org/eclipse/hawkbit/ddi/rest/resource/DataConversionHelper.java

```diff
@@ -118,7 +118,8 @@ static DdiControllerBase fromTarget(final Target target, final Action action,
                 result.add(ControllerLinkBuilder
                         .linkTo(ControllerLinkBuilder.methodOn(DdiRootController.class, tenantAware.getCurrentTenant())
                                 .getControllerBasedeploymentAction(tenantAware.getCurrentTenant(),
-                                        target.getControllerId(), action.getId(), calculateEtag(action)))
+                                        target.getControllerId(), action.getId(), calculateEtag(action),
+                                        Integer.parseInt(DdiRestConstants.NO_ACTION_HISTORY)))
```

Comment:
I think it would be better to put null in here (and change signature to Integer). This is the default value anyhow so no reason to put it into the link.